setup.py

```diff
@@ -308,6 +308,9 @@ def write_version(filename=os.path.join(*[my_dir, "airflow", "git_version"])):
     'dnspython>=1.13.0,<2.0.0',
     'pymongo>=3.6.0',
 ]
+mlflow = [
+    'mlflow>=1.9.1',
+]
```

Comment:
We want to use our own mlflow version, right?

Reply:
Isn't a necessity, the import function is pretty standard. If we do use our version, how do you suggest we incorporate in setup installation?

Reply:
using the git url? Or perhaps pointing to the local filesystem?
I think it is better to be consistent in the long run, no matter how much inconvinience it might cause now.

Reply:
Fair enough, I'll install the package from the master branch of our mlflow repo. Or do you want to have some other branch as the target? A 'wfo-deployment' branch or something of the kind.